src/main/java/com/github/admissionCommittee/util/validate/UserValidatorUtil.java

```diff
@@ -2,35 +2,77 @@
 
 import com.github.admissionCommittee.model.AbstractEntity;
 import com.github.admissionCommittee.model.User;
+import com.github.admissionCommittee.util.validate.org.apache.commons
+        .validator.routines.EmailValidator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.LocalDate;
+import java.util.regex.Pattern;
+
 
 public class UserValidatorUtil extends ValidatorUtil {
+    private static final Logger log = LoggerFactory.getLogger
+            (UserValidatorUtil.class);
+
     @Override
-    public void validateEntity(AbstractEntity entityToValidate) {
+    public void validate(AbstractEntity entityToValidate) {
         User toValidate = (User) entityToValidate;

```

Comment:
class cast here is really bad solution... I suppouse it's better to make your `ValidatorUtil` paramitrise (like ` ValidatorUtil<T extends AbstractEntity>`

Reply:
ValidatorUtil using for many classes's validation.